mmpose/visualization/local_visualizer_3d.py

```diff
@@ -197,11 +201,12 @@ def _draw_3d_instances_kpts(keypoints,
 
                 kpt_color = kpt_color[valid][..., ::-1] / 255.
 
-                ax.scatter(x_3d, y_3d, z_3d, marker='o', c=kpt_color)
+                ax.scatter(x_3d, y_3d, z_3d, marker='o', color=kpt_color)
```

Comment:
Is this modification necessary? The use of `color= kpt_color` seems to be incompatible with the latest version of matplotlib